### ISSUE TITLE
BB Gun

### ISSUE THREAD

**Issue by SirAJ4895** [studio-minus/ppg-bugs-and-requests#1788]
**Feature description**
A BB gun that takes on the appearance of a pistol with an orange tip. Rather than shooting bullets, it shoots a physical BB that isn't too harmful.

**Feature purpose**
I don't have any ideas, but I'm sure the community could find some uses.

**Comment by bibbybabby** [comment]
i think a physical bb will be laggy, maybe a particle projectile will be better?

**Comment by Drogo-man** [comment]
> i think a physical bb will be laggy, maybe a particle projectile will be better?

Not really. We do have crossbows and the wired crystal thing and they aren't laggy at all.

**Comment by bibbybabby** [comment]
> > i think a physical bb will be laggy, maybe a particle projectile will be better?
> 
> Not really. We do have crossbows and the wired crystal thing and they aren't laggy at all.

They aren't the size of 1 pixel, they are sharp and they are semi auto, imagine if a full auto bb gun was added with the physical projectiles

**Comment by DindinYT37** [comment]
Duplicates of or Related to:
* #526
* #994

**Comment by SirAJ4895** [comment]
> > > i think a physical bb will be laggy, maybe a particle projectile will be better?
> > 
> > 
> > Not really. We do have crossbows and the wired crystal thing and they aren't laggy at all.
> 
> They aren't the size of 1 pixel, they are sharp and they are semi auto, imagine if a full auto bb gun was added with the physical projectiles

It would be a semi-automatic

**Comment by bibbybabby** [comment]
> > > > i think a physical bb will be laggy, maybe a particle projectile will be better?
> > > 
> > > 
> > > Not really. We do have crossbows and the wired crystal thing and they aren't laggy at all.
> > 
> > 
> > They aren't the size of 1 pixel, they are sharp and they are semi auto, imagine if a full auto bb gun was added with the physical projectiles
> 
> It would be a semi-automatic

Yeah but still imagine the mess and lag the 1 pixel BB's would bring

**Comment by SirAJ4895** [comment]
I can't imagine how much it would cause since they could just be deleted after coming to a stop

**Comment by bibbybabby** [comment]
> I can't imagine how much it would cause since they could just be deleted after coming to a stop

Fair enough
